TheWarWithin/Priorities/MageFrost.simc

```diff
@@ -1,24 +1,27 @@
+actions.precombat+=/variable,name=time_to_coc,condition=talent.frostfire_bolt.enabled,value=10,value_else=23
 actions.precombat+=/arcane_intellect
 actions.precombat+=/mirror_image
-actions.precombat+=/blizzard,if=active_enemies>=2&talent.ice_caller&!talent.fractured_frost|active_enemies>=3
-actions.precombat+=/frostbolt,if=active_enemies<=2
+actions.precombat+=/blizzard,if=active_enemies>=2&talent.ice_caller&!talent.fractured_frost|active_enemies>=4
+actions.precombat+=/frostbolt,if=active_enemies<=3
 
 actions+=/counterspell
 actions+=/call_action_list,name=cds
-actions+=/run_action_list,name=aoe,strict=1,if=active_enemies>=7|active_enemies>=3&talent.ice_caller
-actions+=/run_action_list,name=ss_cleave,strict=1,if=active_enemies>=2&active_enemies<=3&talent.splinterstorm
-actions+=/run_action_list,name=cleave,strict=1,if=active_enemies>=2&active_enemies<=3
-actions+=/run_action_list,name=ss_st,strict=1,if=talent.splinterstorm
-actions+=/run_action_list,name=st
+actions+=/call_action_list,name=aoe,strict=1,if=active_enemies>=7|active_enemies>=3&!talent.frostfire_bolt|active_enemies>=4
```

Comment:
I'm confused; the first `active_enemies>=7|` is redundant because `|active_enemies>=4` will also be true any time you have 7+ targets.

Is there some grouping to these clauses that is intended?  If the intent is just that you AOE with 3+ targets without Frostfire Bolt talented or 4+ targets with Frostfire Bolt, this simplifies to:

`active_enemies>2&!talent.frostfire_bolt|active_enemies>3`

Note that I changed `>=3` to `>2` and `>=4` to `>3`.  The difference might save a whole nanosecond per year.

Reply:
I think this condition might be changing anyways with the `actions.aoe` change I am submitting

Reply:
that or just a silly entry by the APL writer, will simplify.